pkg/operator/ceph/cluster/osd/config/scheme_test.go

```diff
@@ -178,6 +180,46 @@ func TestSchemeEntryGetPartitionArgs(t *testing.T) {
 	assert.Equal(t, expectedArgs, args)
 
 	logger.Noticef("%+v", args)
+
+	//Explicitly test Bluestore scheme
+	e2 := NewPerfSchemeEntry(Bluestore)
```

Comment:
could we have a helper method to reduce duplication of test code? The "bluestore" test looks the same as the test above it with the "" default bluestore.

Reply:
@travisn fixed.